pkg/action/hooks.go

```diff
@@ -0,0 +1,90 @@
+package action
```

Comment:
`pkg/action` is the package that maps client verbs to logic so that folks who want to import the client logic can do so. In `helm 2.x`, this all lived under `cmd/helm` and was hard to import/reuse. I'm not sure that the hook execution logic should live here. Did you have some thoughts around you wanted to have this logic in `pkg/action` and not `pkg/release`?

Reply:
I ran into some issues with circular dependencies which is why I ended up leaving this here. All of these methods are private to the actions package however so it'll be possible to refactor them out into a public API in the future... this felt like the most minimal change possible at the moment though. :)